starter-api/src/main/resources/META-INF/native-image/io.micronaut.starter.api/micronaut-starter-api/resource-config.json

```diff
@@ -3,6 +3,7 @@
     {"pattern":"\\QMETA-INF/swagger\\E"},
     {"pattern":".*/swagger/micronaut-launch-.*yml"},
     {"pattern":"\\QMETA-INF/swagger/views/rapidoc/index.html\\E"},
-    {"pattern":"\\QMETA-INF/swagger/views/swagger-ui/index.html\\E"}
+    {"pattern":"\\QMETA-INF/swagger/views/swagger-ui/index.html\\E"},
+    {"pattern":"\\select_options.properties\\E"}
```

Comment:
How are you building your native-image?

When I run  (with `sdk use java 21.1.0.r11-grl`)

```
native-image --no-fallback -cp starter-web-netty-2.5.6-SNAPSHOT-all.jar
```

I get

```
[micronaut-starter:99711]    classlist:   1,989.95 ms,  0.94 GB
[micronaut-starter:99711]        (cap):   2,520.20 ms,  1.19 GB
[micronaut-starter:99711]        setup:   4,185.36 ms,  1.19 GB
[micronaut-starter:99711]     analysis:   7,650.95 ms,  1.70 GB
Fatal error:java.util.regex.PatternSyntaxException: Illegal/unsupported escape sequence near index 27
\select_options.properties\E
                                              ^
        at java.base/java.util.regex.Pattern.error(Pattern.java:2027)
        at java.base/java.util.regex.Pattern.escape(Pattern.java:2616)
...
```

Reply:
Damn it! I missed the `Q`. @eahrold please try with 
`{"pattern":"\\Qselect_options.properties\\E"}` and if it works, please push the commit with the fix. Sorry about it

Reply:
Pushed escape sequence fix, but it still doesn't look like it's picking up the resource for some reason.

http://localhost:8080/select-options

```
    "test": {
        "options": [
            {
                "description": "Junit",
                "name": "junit",
                "value": "JUNIT",
                "label": "Junit",
                "_links": {}
            },
```